rskj-core/src/main/java/co/rsk/peg/BridgeStorageProvider.java

```diff
@@ -592,6 +595,29 @@ private void saveCoinbaseInformations() {
             safeSaveToRepository(getStorageKeyForCoinbaseInformation(blockHash), data, BridgeSerializationUtils::serializeCoinbaseInformation));
     }
 
+    public Optional<Long> getReceiveHeadersLastTimestamp() {
+        if (activations.isActive(RSKIP200)) {
+            Optional<Long> optionalReceiveHeadersTimestamp = safeGetFromRepository(
+                    RECEIVE_HEADERS_TIMESTAMP,
+                    BridgeSerializationUtils::deserializeOptionalLong
+            );
+            return optionalReceiveHeadersTimestamp;
```

Comment:
In order to avoid code smell, return directly: 

`safeGetFromRepository(RECEIVE_HEADERS_TIMESTAMP,BridgeSerializationUtils::deserializeOptionalLong);`